x-pack/plugins/cloud_security_posture/public/pages/rules/index.tsx

```diff
@@ -37,6 +40,19 @@ export const Rules = ({ match: { params } }: RouteComponentProps<PageUrlParams>)
   const pageProps: KibanaPageTemplateProps = useMemo(
     () => ({
       pageHeader: {
+        alignItems: 'bottom',
+        rightSideItems: [
+          <EuiButtonEmpty
+            iconType="gear"
+            size="xs"
+            href={http.basePath.prepend(pagePathGetters.edit_integration(params).join(''))}
+          >
+            <FormattedMessage
+              id="xpack.csp.rules.manageIntegrationButtonLabel"
+              defaultMessage="Manage Integration"
+            />
+          </EuiButtonEmpty>,
+        ],
```

Comment:
the new design shows this button on the page header (above the header border) 
so this button had to move here from the page body

we position it on the right using `rightSideItems` and `alignItems: bottom`, it's OK - but later on we might fine tune it to fit perfectly

Reply:
lgtm